app/components/DarkModeToggle.js

```diff
@@ -26,7 +26,7 @@ const DarkModeToggle = ({ className }) => {
         <BiToggleLeft size="2rem" className="inline" />
       )}
       <span className="mx-1">
-        Dark<span className="lg:hidden"> Mode</span>
+        Donker<span className="lg:hidden"> Modus</span>
```

Comment:
Wat als je de `e` van Donker(e) nog binnen de `<span>` zet? Ik weet niet of we dit zelfs moeten vertalen. Misschien is Dark Mode al genoeg ingeburgerd?